components/scheduler/src/Scheduler.svelte

```diff
@@ -26,6 +25,8 @@
   export let booking_label: string;
   export let event_title: string;
   export let event_description: string;
+  export let event_location: string;
+  export let event_conferencing: string;
```

Comment:
Do we want to accept the whole conferencing object here instead of just the url?

Reply:
@ozsivanov I'm super on the fence about it at the moment. My gut tells me to keep it simple for right now (I think I'm going to do some url-detection to determine provider, rather than ask the user for it, for example)

I think I'd like to keep it just URL for now, and when someone inevitably asks us for a `pin` or `phone` fields, we can worry about it then.

Reply:
I like it keeping it as a URL for now too